Core/include/Acts/Vertexing/AdaptiveGridDensityVertexFinder.ipp

```diff
@@ -6,24 +6,24 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-template <typename vfitter_t>
-auto Acts::AdaptiveGridDensityVertexFinder<vfitter_t>::find(
+inline auto Acts::AdaptiveGridDensityVertexFinder::find(
     const std::vector<InputTrack>& trackVector,
-    const VertexingOptions& vertexingOptions, State& state) const
+    const VertexingOptions& vertexingOptions, IVertexFinder::State& state) const
     -> Result<std::vector<Vertex>> {
+  auto& thisState = state.as<State>();
```

Comment:
alternatively you could rename the param and leave the concrete state with the same name. that might be diff friendlier

Reply:
Changed.